lib/project/result-view.coffee

```diff
@@ -44,11 +44,61 @@ class ResultView extends View
       @hide()
     else
       @show()
+      @addContextToMatches(@filePath, matches)
       for match in matches
         @matches.append(new MatchView(@model, {@filePath, match}))
 
     @matches.children().eq(selectedIndex).addClass('selected') if selectedIndex > -1
 
+  addContextToMatches: (filePath, matches) ->
+    CONTEXT_LINES = atom.config.get('find-and-replace.searchContextExtraLines')
+    return if CONTEXT_LINES is 0
+    content = fs.readFileSync(filePath).toString()
+    lines = content.split('\n')
```

Comment:
Fixed in https://github.com/atom/find-and-replace/pull/769/commits/3d7e8cd386732ae4737e1de011367585a4399d12